static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-epic.html',
```

Comment:
They're both issues for different reasons - the combined size will mean that the JS is larger coming down the wire and hence take longer and the more files there are, the more work the module loader has to do when it is processing the JS file.